catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlPreIngestPluginTest.java

```diff
@@ -66,24 +64,22 @@ protected Subject getSubject() {
   }
 
   @Test
-  public void testCreateShouldIgnoreNonShareableMetacards() throws Exception {
-    ShareableMetacardPreIngestPlugin plugin = makePlugin();
+  public void testCreateShouldIgnoreNonACLMetacards() throws Exception {
```

Comment:
I think by convention we don't capitalize acronyms (`testCreateShouldIgnoreNonAclMetacards`)